rest-api-spec/src/main/resources/rest-api-spec/api/search.json

```diff
@@ -76,6 +76,10 @@
           "type" : "string",
           "description" : "Specify the node or shard the operation should be performed on (default: random)"
         },
+        "prefixed_name": {
+          "type" : "boolean",
+          "description" : "Specify whether aggregation names should be prefixed by their respective types in the response"
+        },
```

Comment:
I think it makes sense that msearch also has this parameter so I updated the code.

For search template APIs, I don't know. I guess it makes sense, but I'd like to keep this PR short - could we deal with this when we'll add parsing methods for Search Template APIs?